GGProject/workflow/apps.py

```diff
@@ -3,4 +3,4 @@
 
 class WorkflowConfig(AppConfig):
     name = 'workflow'
-    verbose_name = 'Objects'
+    verbose_name = 'Objects'
```

Comment:
This file has no new line at the end. Thats what that red icon is showing. Its pretty standard to have a new line at the end of files. I believe it's also a readability thing